server/migrations/1620672944865_update-triggers.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable camelcase */
+
+const { collections } = require('../db');
+
+exports.up = async (pgm) => {
+  await pgm.dropTrigger(collections.EMPLOYER_SITES, 'update_postal_code_s', { ifExists: true });
+  await pgm.createTrigger(collections.EMPLOYER_SITES, 'update_postal_code_s', {
+    when: 'BEFORE',
+    operation: ['INSERT', 'UPDATE OF body'],
+    language: 'plpgsql',
+    level: 'ROW',
+    replace: true,
+  },
+  `
+  DECLARE
+   loc record;
+   participant record;
+  BEGIN
+    IF NEW.body->>'postalCode' = OLD.body->>'postalCode' THEN
+      RETURN NEW;
+    END IF;
+
+    SELECT longitude::text,latitude::text INTO loc FROM geocodes WHERE REPLACE(postal_code, ' ', '')=REPLACE(NEW.body->>'postalCode', ' ', '');
+
+    IF NOT FOUND THEN
+      SELECT longitude::text,latitude::text INTO loc FROM geocodes WHERE SUBSTRING(REPLACE(postal_code, ' ', ''),1,5)=SUBSTRING(REPLACE(NEW.body->>'postalCode', ' ', ''),1,5);
+    END IF;
+
+    IF NOT FOUND THEN
+      IF NEW.body->>'location' IS NOT NULL THEN
+        NEW.body = NEW.body::jsonb - 'location';
+        DELETE FROM participants_distance where site_id = OLD.id;
```

Comment:
I think this should be referencing `body->>'siteId'`, no? For participants `id` is used but for sites we're not using the default `id` column but rather the `siteId` in the JSONB.

Reply:
This is the regular `site_id` column on `participants_distance` not a jsonb right?

Reply:
This is the regular site_id column on participants_distance not a jsonb right?

Reply:
Ah yes, this is valid. I really don't have a strong understanding of the difference between how id and siteId are used in sites. I think that you're probably right to suggest this correction here, I'll update